docs/tools/reset_config.md

```diff
@@ -0,0 +1,44 @@
+## Send a configuration message to a specific device
+
+The `reset_config` tool can be used to send a configuration message to a specific device.
+
+The `reset_config` message can be invoked from the `udmi` folder in the following way:
```

Comment:
There's no "reset config" message as such -- it's a standard "config update" message that is used to reset the config. So I'd change this to just "command can be invoked." The distinction is important (technical documentation) because otherwise people will go off looking for a specific message type (which doesn't exist so they'd never find it!).

Reply:
Change done as suggested.